bamboo/integration_tests/common_code.py

```diff
@@ -1,269 +0,0 @@
-import sys
```

Comment:
We don't want to keep any of the old integration tests? These two new tests will be sufficient?

Reply:
I think ResNet-18 and ResNet-50 do a good job validating our image classification functionality and they use up all the available time. We'll need to make these tests smaller if we also want to test autoencoders or non-vision models.

Reply:
If you think we don't *need* the autoencoder and non-vision models tested then I suppose just these two tests are good then.

Reply:
Just jumping in here but we are going to want non-vision models tested and autoencoders.  Ideally, we will have integrated tests for every LBANN application.  Time will be an issue here.